services/registry/docker-compose.yml.j2

```diff
@@ -115,11 +115,12 @@ services:
         parallelism: 1
       placement:
         constraints:
-          - node.labels.ops==true
```

Comment:
Chore:
- use `.master.` docker compose file or j2 instead of negative labels
-  spread-constraint